src/sentry/db/models/fields/hybrid_cloud_foreign_key.py

```diff
@@ -0,0 +1,95 @@
+"""
+A 'foreign key' which is not enforced in the local database, but triggers eventually consistent delete work in the
+presence of RegionTombstone ControlTombstone model objects through the tasks/deletion/hybrid_cloud.py logic.
+
+It's main purpose to support columns in, say, region silos that refer to User or Integration objects (conversely
+also columns in the control silo that point to, say, organization) that do not actually exist in the local database,
+or even the local network.
+
+Functionally, this field is just a dumb BigIntegerField.  It does nothing, at the database level, that a foreign key
+normally does.  It does not provide any constraints: this means, for instance, you should absolutely expect identifiers
+in this column that do not necessarily exist.  Eventually, if the related object is deleted and processed correctly,
```

Comment:
I'm beginning to wonder if I should just keep chatgpt running and completing my documentation these days because I am absolute trash at maintaining comments.